### PR TITLE
[Projects] Fix creation of Razor Class lib projects

### PR DESCRIPTION
**Pull request by rodrmoya**
When creating a new Razor Class lib project, this exception was being
thrown:

ERROR [2019-08-13 13:54:48Z]: The project could not be created
System.MissingMethodException: Default constructor not found for type MonoDevelop.Projects.FrameworkReference
  at System.RuntimeType.CreateInstanceMono (System.Boolean nonPublic, System.Boolean wrapExceptions) [0x00066] in /Users/builder/jenkins/workspace/build-package-osx-mono/2019-06/external/bockbuild/builds/mono-x64/mcs/class/corlib/ReferenceSources/RuntimeType.cs:177
  at System.RuntimeType.CreateInstanceSlow (System.Boolean publicOnly, System.Boolean wrapExceptions, System.Boolean skipCheckThis, System.Boolean fillCache) [0x00009] in /Users/builder/jenkins/workspace/build-package-osx-mono/2019-06/external/bockbuild/builds/mono-x64/mcs/class/corlib/ReferenceSources/RuntimeType.cs:155
  at System.RuntimeType.CreateInstanceDefaultCtor (System.Boolean publicOnly, System.Boolean skipCheckThis, System.Boolean fillCache, System.Boolean wrapExceptions, System.Threading.StackCrawlMark& stackMark) [0x00027] in /Users/builder/jenkins/workspace/build-package-osx-mono/2019-06/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/rttype.cs:5729
  at System.Activator.CreateInstance (System.Type type, System.Boolean nonPublic, System.Boolean wrapExceptions) [0x00039] in /Users/builder/jenkins/workspace/build-package-osx-mono/2019-06/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/activator.cs:206
  at System.Activator.CreateInstance (System.Type type, System.Boolean nonPublic) [0x00000] in /Users/builder/jenkins/workspace/build-package-osx-mono/2019-06/external/bockbuild/builds/mono-x64/mcs/class/referencesource/mscorlib/system/activator.cs:190
  at MonoDevelop.Projects.Project.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00034] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:1052
  at MonoDevelop.Projects.DotNetProject.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00032] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/DotNetProject.cs:266
  at MonoDevelop.Projects.Project+DefaultMSBuildProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:4861
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.ProjectExtension.OnCreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:222
  at MonoDevelop.Projects.Project.CreateProjectItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated item) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:1042
  at MonoDevelop.Projects.Project.ReadItem (MonoDevelop.Projects.MSBuild.IMSBuildItemEvaluated buildItem, MonoDevelop.Core.Collections.LookupTable`2[TKey,TElement] lookupItems) [0x00172] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:3206
  at MonoDevelop.Projects.Project.LoadProjectItems (MonoDevelop.Projects.MSBuild.MSBuildProject msproject, MonoDevelop.Projects.ProjectItemFlags flags, System.Collections.Generic.HashSet`1[T] loadedItems) [0x0010f] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:3117
  at MonoDevelop.Projects.Project.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x0006b] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:2780
  at MonoDevelop.Projects.DotNetProject.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/DotNetProject.cs:2068
  at MonoDevelop.Projects.Project+DefaultMSBuildProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:4919
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.GtkCore.GuiBuilder.GtkProjectServiceExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.GtkCore/MonoDevelop.GtkCore.GuiBuilder/GtkProjectServiceExtension.cs:24
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Autotools.MakefileProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/addins/MonoDevelop.Autotools/MakefileProjectServiceExtension.cs:82
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.SdkProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00018] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/SdkProjectExtension.cs:102
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.SharedAssetsProjects.SharedAssetsProjectMSBuildExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects.SharedAssetsProjects/SharedAssetsProjectMSBuildExtension.cs:40
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.SdkProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00018] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/SdkProjectExtension.cs:102
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.SdkProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00018] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/SdkProjectExtension.cs:102
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.ProjectExtension.OnReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00001] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectExtension.cs:146
  at MonoDevelop.Projects.Project.ReadProject (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.MSBuild.MSBuildProject msproject) [0x00068] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:2629
  at MonoDevelop.Projects.Project.LoadAsync (MonoDevelop.Core.ProgressMonitor monitor) [0x0014f] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/Project.cs:437
  at MonoDevelop.Projects.MSBuild.MSBuildProjectService.LoadItem (MonoDevelop.Core.ProgressMonitor monitor, System.String fileName, MonoDevelop.Projects.MSBuild.MSBuildFileFormat expectedFormat, System.String typeGuid, System.String itemGuid, MonoDevelop.Projects.SolutionLoadContext ctx) [0x00238] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects.MSBuild/MSBuildProjectService.cs:405
  at MonoDevelop.Projects.SdkProjectReader.LoadSolutionItem (MonoDevelop.Core.ProgressMonitor monitor, MonoDevelop.Projects.SolutionLoadContext ctx, System.String fileName, MonoDevelop.Projects.MSBuild.MSBuildFileFormat expectedFormat, System.String typeGuid, System.String itemGuid) [0x00068] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/SdkProjectReader.cs:102
  at MonoDevelop.Projects.ProjectService+<>c__DisplayClass25_0.<ReadSolutionItem>b__0 () [0x0016e] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectService.cs:126
  at MonoDevelop.Projects.ProjectService.ReadSolutionItem (MonoDevelop.Core.ProgressMonitor monitor, System.String file) [0x00049] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Projects/ProjectService.cs:111
  at MonoDevelop.Ide.Templates.MicrosoftTemplateEngineProjectTemplatingProvider.ProcessTemplate (MonoDevelop.Ide.Templates.SolutionTemplate template, MonoDevelop.Ide.Projects.NewProjectConfiguration config, MonoDevelop.Projects.SolutionFolder parentFolder) [0x003e8] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Templates/MicrosoftTemplateEngineProjectTemplatingProvider.cs:109
  at MonoDevelop.Ide.Templates.TemplatingService.ProcessTemplate (MonoDevelop.Ide.Templates.SolutionTemplate template, MonoDevelop.Ide.Projects.NewProjectConfiguration config, MonoDevelop.Projects.SolutionFolder parentFolder) [0x0005a] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Templates/TemplatingService.cs:176
  at MonoDevelop.Ide.Projects.NewProjectDialogController.CreateProject () [0x0026d] in /Users/rodrigo/Projects/xamarin/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Projects/NewProjectController.cs:788
Thread started: <Thread Pool> #52

### CODE REVIEW

**Review by mrward — APPROVED** [review]

**Comment by rodrmoya** [comment]
@monojenkins backport release-8.3

**Review by Therzok — APPROVED** [review]

**Review by bratsche — APPROVED** [review]

**Comment by rodrmoya** [comment]
@monojenkins backport release-8.3

File: main/src/addins/MonoDevelop.AspNetCore/MonoDevelop.AspNetCore.Tests/MonoDevelop.AspNetCore.Tests.csproj [hunk 1]
```diff
@@ -72,6 +73,10 @@
       <Name>IdeUnitTests</Name>
       <Private>False</Private>
     </ProjectReference>
+    <ProjectReference Include="..\..\MonoDevelop.DotNetCore\MonoDevelop.DotNetCore.Tests\MonoDevelop.DotNetCore.Tests.csproj">
+      <Project>{7D3073E9-F52D-4E14-B988-5316DC6FD0BF}</Project>
+      <Name>MonoDevelop.DotNetCore.Tests</Name>
```
Comment (Therzok) [inline main/src/addins/MonoDevelop.AspNetCore/MonoDevelop.AspNetCore.Tests/MonoDevelop.AspNetCore.Tests.csproj]: Private?

Comment (rodrmoya) [reply main/src/addins/MonoDevelop.AspNetCore/MonoDevelop.AspNetCore.Tests/MonoDevelop.AspNetCore.Tests.csproj]: As suggested by Matt, removed the dependency, as it's not needed

**Comment by rodrmoya** [comment]
> Does this fail if the FrameworkReference does not have a constructor? Just wondering if the solution needs to be restored for this to fail or not. I have no idea 😄

yes, without the constructor the test fails, so just loading the project shows it

**Comment by rodrmoya** [comment]
@monojenkins backport release-8.3

**Pull request merged**